desci-repo/src/db/index.ts

```diff
@@ -6,7 +6,7 @@ const { Pool } = pg;
 console.log('DB', process.env.DATABASE_URL);
 
 export const pool = new Pool({
-  connectionString: process.env.DATABASE_URL,
+  connectionString: process.env.DATABASE_URL || 'postgresql://walter:white@host.docker.internal:5433/boilerplate',
```

Comment:
I personally think it's nicer to have things fail when configuration is unset, instead of trying to run with arbitrary config :)

Reply:
oh yeah gotta remove that, added it for running test outside docker container lol